src/container/EdgeRenderer/index.tsx

```diff
@@ -218,49 +222,156 @@ const EdgeRenderer = (props: EdgeRendererProps) => {
+              connectionHandleType={connectionHandleType!}
+              connectionPositionX={connectionPosition.x}
+              connectionPositionY={connectionPosition.y}
+              transform={transform}
+              connectionLineStyle={connectionLineStyle}
+              connectionLineType={connectionLineType}
+              connectionSourceOffsetX={connectionSourceOffsetX}
+              connectionSourceOffsetY={connectionSourceOffsetY}
+              isConnectable={nodesConnectable}
+              CustomConnectionLineComponent={connectionLineComponent}
+            />
+          </g>
+        </svg>
+      )}
+      {edges.map((edge: Edge) => {
+        return (
+          <svg
             key={edge.id}
-            edge={edge}
-            props={props}
-            nodes={nodes}
-            selectedElements={selectedElements}
-            elementsSelectable={elementsSelectable}
-            transform={transform}
             width={width}
             height={height}
-            onlyRenderVisibleElements={onlyRenderVisibleElements}
-          />
-        ))}
-        {renderConnectionLine && (
-          <ConnectionLine
-            nodes={nodes}
-            connectionNodeId={connectionNodeId!}
-            connectionHandleId={connectionHandleId}
-            connectionHandleType={connectionHandleType!}
-            connectionPositionX={connectionPosition.x}
-            connectionPositionY={connectionPosition.y}
-            transform={transform}
-            connectionLineStyle={connectionLineStyle}
-            connectionLineType={connectionLineType}
-            connectionSourceOffsetX={connectionSourceOffsetX}
-            connectionSourceOffsetY={connectionSourceOffsetY}
-            isConnectable={nodesConnectable}
-            CustomConnectionLineComponent={connectionLineComponent}
-          />
-        )}
-      </g>
-    </svg>
+            className="react-flow__edges"
+            style={{
+              zIndex: calculateZIndexes(
+                props.mostRecentlyTouchedSceneIds,
+                edge.target,
+                edge.source,
+                Boolean(renderConnectionLine)
+              ),
+            }}
+          >
+            <MarkerDefinitions color={arrowHeadColor} />
+            <g>
+              <Edge
+                edge={edge}
+                props={props}
+                nodes={nodes}
+                selectedElements={selectedElements}
+                elementsSelectable={elementsSelectable}
+                transform={transform}
+                width={width}
+                height={height}
+                onlyRenderVisibleElements={onlyRenderVisibleElements}
+              />
+            </g>
+          </svg>
+        );
+      })}
+    </div>
   );
 };
 
-EdgeRenderer.displayName = 'EdgeRenderer';
+BaseEdgeRenderer.displayName = 'EdgeRenderer';
 
-export default memo(EdgeRenderer);
+export const EdgeRenderer = memo(BaseEdgeRenderer);
```

Comment:
Changed this default export in order to `export interface EdgeRendererProps` above